resource-packs/Catppuccin UI/create_flavors.py

```diff
@@ -141,50 +158,86 @@ def main():
             f'\'import_language_files.py\' script in the \'lang\' folder.')
         return
 
-    # Get all accent colors from the catppuccin palette library
+    # ask the user for flavor and accent color if not specified
+    flavor_names = [flavor.name for flavor in PALETTE]
+    while True:
+        if selected_flavor is not None:
+            break
+        selected_flavor = input(
+            '-----------------------------------------------------------------------\n'
+            'Enter which flavor to create. Leave blank for default (all)\n'
+            f'Available flavors: {flavor_names}\n').capitalize()
+        if selected_flavor not in flavor_names + ['Frappe', ""]:
+            print(f'ERROR: Flavor \'{selected_flavor}\' not found! Choose from the available flavors.')
+            selected_flavor = None
+    
     accent_colors = [color.name for color in PALETTE.mocha.colors if color.accent == True]
+    while True:
+        if selected_accent is not None:
+            break
+        selected_accent = input(
+            '-----------------------------------------------------------------------\n'
+            'Enter which accent to create. Leave blank for default (all)\n'
+            f'Available accents: {accent_colors}\n').capitalize()
+        if selected_accent not in accent_colors + [""]:
+            print(f'ERROR: accent \'{selected_accent}\' not found! Choose from the available accents.')
+            selected_accent = None
+    # Get all flavors and accent colors from the catppuccin palette library if not specified.
+    flavor_names = [selected_flavor] if selected_flavor else flavor_names
```

Comment:
If flavor argument is not specified the frappe flavor is skipped because "Frappe" is not in the flavor_names list
```suggestion
    flavor_names = [selected_flavor] if selected_flavor else flavor_names + ['Frappe'] # Frappe accented e workaround.
```

Reply:
I just tested it without any arguments and the Frappe from the library's palette object is still there, the other "Frappe" was added to allow inputs without the accent to also be accepted.

Reply:
When actually generating the resourcepack for all flavors and the script is generating frappe the flavor variable is assigned to "Frappe" on [line 216](https://github.com/madhavarun/minecraft/blob/0f71163a1e16af3f3edd68493c70f7af71a09c22/resource-packs/Catppuccin%20UI/create_flavors.py#L216). After that on [line 220](https://github.com/madhavarun/minecraft/blob/0f71163a1e16af3f3edd68493c70f7af71a09c22/resource-packs/Catppuccin%20UI/create_flavors.py#L220) there is check if the flavor variable is in the flavor_names list and it is not, because just few lines earlier the flavor was assigned to "Frappe" instead of "Frappé" which is in the flavor_names list.

Reply:
OH yeah that makes sense yeah I can change that then. Thanks for pointing it out